qt/aqt/main.py

```diff
@@ -960,6 +962,7 @@ def wrap_on_updates_installed(log: list[DownloadLogEntry]) -> None:
                 requested_by_user=False,
             )
             self.pm.set_last_addon_update_check(int_time())
+            self.pm.set_last_run_version()
```

Comment:
This might cause confusion down the line. We might start using the last run version for something other than add-on checks (eg, to upgrade the profile metadata), and might add some option in the future to disable the check for add-on updates, not realizing it will also prevent the last run version from being updated. Maybe it might be better to move this up into the onsuccess call, and pass a version_changed option into maybe_check_for_addon_updates(), or something like that?

Reply:
Good point, though I'd also like to avoid setting and reading the value at different places. Then you'd always have to double-check whether you're reading the old or the new value.
What do you think of the new solution?

Reply:
Much nicer - now code that doesn't run immediately can also get at the previous version too. Thanks Rumo!